src/components/radio-button/radiobutton.test.js

```diff
@@ -316,29 +298,23 @@ context("Testing RadioButton component", () => {
       <RadiobuttonComponent error="Error has occurred" />
     );
 
-    radiobuttonIcon()
-      .should("have.attr", "aria-label", "Error has occurred")
-      .should("have.attr", "data-element", "error");
+    radiobuttonIcon().should("have.attr", "data-element", "error");
```

Comment:
Similar to Checkbox for the changed tests in this file

```suggestion
    radiobuttonIcon().should("have.attr", "data-element", "error");
    tooltipPreview().should("have.text", "Error has occurred");
```